packages/itwin/reports-config-widget/src/widget/components/ReportMappings.tsx

```diff
@@ -342,6 +292,41 @@ export const ReportMappings = ({ report, goBack }: ReportMappingsProps) => {
         }}
         refresh={refresh}
       />
+      <div id="action">
```

Comment:
Is this `id` needed? Is this `div` element needed?

Reply:
`id` has been removed